src/main/java/com/univapay/sdk/types/CardBrand.java

```diff
@@ -19,11 +27,14 @@ public enum CardBrand {
   @SerializedName("maestro")
   MAESTRO("^(5018|5020|5038|6020|6304|6703|6759|676[1-3])\\d*", 12, 19, 3, "maestro"),
   @SerializedName("unionpay")
-  UNIONPAY("^62\\d*", 16, 19, 3, "unionpay"),
+  UNIONPAY("^(62[0-9]{14,17})$", 16, 19, 3, "unionpay"),
+
+  @SerializedName("private_label")
+  PRIVATE_LABEL(12, 19, 4, "private_label"),
```

Comment:
Not sure but security code length might be 3??

Reply:
Yeah, not sure about that too, I will mark as 3, but the field itself is not used for anything in the SDK.
I will mark as deprecated along with the regexp and validation feature.